src/fm/test/report.clj

```diff
@@ -38,8 +38,7 @@
       :total
       (partial update-result-total :pass num-tests time-elapsed-ms)))))
 
-(defmethod group-result-data-reducer
-  :fail
+(defmethod group-result-data-reducer :fail
```

Comment:
We may want to add a `:default` case here that just passes along `acc` unchanged, or some other default behavior.

Reply:
Good call.